client/txn.go

```diff
@@ -295,6 +295,10 @@ func (txn *Txn) ReverseScan(begin, end interface{}, maxRows int64) ([]KeyValue,
 	return txn.scan(begin, end, maxRows, true)
 }
 
+// TODO(pmattis): Txn.ReverseScan is neither used or tested. Silence unused
+// warning.
+var _ = (*Txn)(nil).ReverseScan
```

Comment:
And I was trying to avoid doing an substantive change in this PR. We should certainly add a test. I'll file issues for these tests when this PR goes in.